app/src/main/kotlin/at/bitfire/davdroid/servicedetection/HomeSetRefresher.kt

```diff
@@ -159,4 +161,27 @@ class HomeSetRefresher @AssistedInject constructor(
         }
     }
 
+    /**
+     * Evaluates whether a response is personal or not.
+     * It takes the [Owner] property from the response, and compares its value against [service]'s [Service.principal].
+     *
+     * If either one of those is not set (`null`), this function returns `null`.
```

Comment:
Sorry, I did not check properly. Kdoc needs to be updated as well now (function does not return null anymore, but rather `false`).